docs/Project.toml

```diff
@@ -16,3 +18,9 @@ Literate = "98b081ad-f1c9-55d3-8b20-4c87d4299306"
 MultiComponentFlash = "35e5bd01-9722-4017-9deb-64a5d32478ff"
 NetworkLayout = "46757867-2c16-5918-afeb-47bfcb05e46a"
 Optim = "429524aa-4258-5aef-a3af-852621145aeb"
+Optimisers = "3bd65402-5787-11e9-1adc-39752487f4e2"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
Does not seem to be used? Loaded in example but there it looks like it is Makie?

Reply:
Plots was used for simple line plots of rel perm curves and losses. But probably best to not mix Plots and GLMakie in the same script. Remove the Plots dependency now.